README.md

```diff
@@ -927,6 +927,12 @@ Here is an example configuration of a sink:
   # The timezone used for the info file
   timezone = "Africa/Johannesburg"
   
+  # Setup Enceladus main class and command line template if you want to run it from Pramen
+  enceladus.run.main.class = "za.co.absa.enceladus.standardization_conformance.StandardizationAndConformanceJob"
+  # Command line template for Enceladus
+  # You can use the following variables: @datasetName, @datasetName, @datasetVersion, @infoDate, @infoVersion, @rawPath, @rawFormat.
+  enceladus.command.line.template = "--dataset-name @datasetName --dataset-version @datasetVersion --report-date @infoDate --menas-auth-keytab menas.keytab --raw-format @rawFormat"
```

Comment:
How do you solve for format-specific options?

Reply:
Users can override `enceladus.command.line.template` in the configuration of a source and insert any options they want. 
Most of the time it will be `parquet` i think when ingesting using Pramen.

Reply:
Thank you